run.sh

```diff
@@ -79,6 +79,8 @@ if [[ ! -f /home/$SPICE_USER/.eyeosConfigured ]]; then
 	touch /home/$SPICE_USER/.eyeosConfigured
 fi
 
+chown -R 1000:1000 /home/$SPICE_USER/.config/libreoffice
```

Comment:
This recursive `chown` (and others present in `run.sh`) are problematic because of latency of the dav mount (files are thousands of kms away from the container). It is possible to avoid `chown`ing anything?